code-studio/src/js/components/Grid.jsx

```diff
@@ -82,7 +159,11 @@ var Grid = React.createClass({
           row={x}
           col={y}
           selected={selected}
-          onSelectionChange={this.props.onSelectionChange}
+          selecting={this.isSelecting(x, y)}
```

Comment:
If this property just describes a style change (from the `Cell`'s point of view), maybe it should be named something like `highlight` or `dragHighlight` instead of `selecting`.